routing/traffic_stash.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "routing/num_mwm_id.hpp"
+
+#include "traffic/traffic_cache.hpp"
+#include "traffic/traffic_info.hpp"
+
+#include "indexer/mwm_set.hpp"
+
+#include <unordered_map>
+
+namespace routing
+{
+class TrafficStash final
+{
+public:
+  TrafficStash(traffic::TrafficCache const & source) : m_source(source) {}
+
+  traffic::TrafficInfo::Coloring const * Get(NumMwmId numMwmId) const
```

Comment:
Why not to return shared_ptr?
```c++
auto if = m_mwmToTraffic.find(numMwmId);
if (it == cend())
  return shared_ptr<>();
return it->second;
```
Raw pointers usually raise a question in the developer's head - "who is the owner of this thing??!"